src/fswAlgorithms/attDetermination/sunlineSuKF/sunlineSuKF.c

```diff
@@ -132,8 +132,10 @@ void Reset_sunlineSuKF(SunlineSuKFConfig *configData, uint64_t callTime,
                                (int32_t) configData->numStates, configData->sQnoise);
     mTranspose(configData->sQnoise, configData->numStates,
                configData->numStates, configData->sQnoise);
-
-    configData->cssSensorInBuffer = CSSArraySensorMsg_C_read(&configData->cssDataInMsg);
+    
+    if (CSSArraySensorMsg_C_isWritten(&configData->cssDataInMsg)){
+        configData->cssSensorInBuffer = CSSArraySensorMsg_C_read(&configData->cssDataInMsg);
+    }
```

Comment:
We need to zero `cssSensorInBuffer` if no message is read.

Reply:
Added the message zero'ing.